app/pages/docs/file-structure.mdx

```diff
@@ -90,47 +90,48 @@ sidebar_label: File Structure
+  - `pages/` weergeeft een React component op een URL. Volgt dezelfde semantiek als de Next.js `pages/api/` map.
+  - `api/` geeft een Node.js request handler vrij op de URL. Volgt dezelfde semantiek als de Next.js `pages/api/` map.
+  - `queries/` en `mutations/` zijn voor je Blitz queries en mutations. Elke query en mutation wordt gekoppeld aan een URL die overeenkomt met het bestandspad.
 
 #### `db`
 
-Contains database configuration, schema, and migration files. `db/index.js` exports your initialized database client so you can use it anywhere in your app.
+Bevat de databaseconfiguratie, schema en migratiebestanden. `db/index.js` exporteert je geïnitialiseerde database client zodat je deze overal in jouw app kan gebruiken.
 
 #### `integrations`
 
-Contains third-party integration code. Ex: `auth0.js`, `twilio.js`, etc. These files are a good place to instantiate a client with shared configuration.
+Bevat integratiecode van derden. Bijvoorbeeld: `auth0.js`, `twilio.js`, etc. Deze bestanden zijn een goede plaats om een client met gedeelde configuratie te instantiëren
 
 #### `pages`
 
-The top level `pages` folder and all nested `pages` folders inside `app` are merged together at build time. The build will fail if the same route is defined in two places.
+De `pages` map op het hoogste niveau map en alle geneste `pages` mappen binnen `app` worden samengevoegd op build time. De build zal mislukken als dezelfde route op twee plaatsen is gedefinieerd.
 
-- This top level pages folder is optional.
-- Has the same semantics as the Next.js `pages` folder. All files in here are mapped to the url corresponding to their file paths.
-- While you can place any route files here, we recommend placing route files inside `app`. But if you want, you can instead place all your route files in this top level folders instead of being spread out in various folders inside `app`
+- De `pages` map op het hoogste niveau is optioneel.
+- Heeft dezelfde semantiek als de Next.js `pages` map. Alle bestanden hierin worden gekoppeld aan een URL die overeenkomt met hun bestandspaden.
+- Hoewel je alle routebestanden hier kan plaatsen, raden wij aan om deze in `app` te plaatsen. Maar je kan ook al jouw routebestanden in deze map op het hoogste niveau plaatsen in plaats van ze te verspreiden over de verschillende 
+mappen binnen `app`.
 
 #### `api`
 
-Same as Next.js `pages/api` folder, but not nested inside pages.
+Hetzelfde als de Next.js `pages/api` map, maar niet genest in `pages`.
 
-And like the `pages` folder, the top level `api` folder and all nested `api` folders inside `app` are merged together at build time.
+En net zoals de `pages` map, worden de `api` map op het hoogste niveau en alle geneste `api` mappen samengevoegd op build time.
 
 #### `public`
 
-All files in here are served statically from your app's root URL
+Alle bestanden in deze map worden statisch geserveerd vanuit de root URL van je app.
```

Comment:
Ik vind _geserveerd_ een beetje raar klinken. Kunnen we dit misschien in het Engels houden? Iets als: _geserved_, als dat bestaat? :)